src/components/fields/CheckboxField.tsx

```diff
@@ -48,7 +48,7 @@ export default function CheckboxField({
         </Box>
         <Box flexShrink={1}>
           <AccentColorProvider color={customColor || action}>
-            <Text color={isChecked ? 'accent' : 'secondary80 (Deprecated)'} size="16px / 22px (Deprecated)" weight="bold">
+            <Text color={isChecked ? 'accent' : 'secondary80 (Deprecated)'} size="15pt" weight="bold">
```

Comment:
for L2s this was overflowing onto the next line for Android so that's why I made this change for context. Not a super good fix but it's a patch

Reply:
I wonder if this is somehow affecting iOS. Maybe safer to wrap in a platform check and keep ios the same?